color_channel_swapper.py

```
@@ -0,0 +1,95 @@
+        self.load_btn.pack(pady=5)
+
+        self.channel_frame = tk.Frame(self.root)
+        self.channel_frame.pack(pady=5)
+        tk.Label(self.channel_frame, text="Swap").pack(side=tk.LEFT)
+        self.channel1 = tk.StringVar(value="R")
+        self.channel2 = tk.StringVar(value="G")
+        tk.OptionMenu(self.channel_frame, self.channel1, "R", "G", "B").pack(side=tk.LEFT)
+        tk.Label(self.channel_frame, text="with").pack(side=tk.LEFT)
+        tk.OptionMenu(self.channel_frame, self.channel2, "R", "G", "B").pack(side=tk.LEFT)
+
+        self.swap_btn = tk.Button(self.root, text="Swap Channels", command=self.swap_channels, state=tk.DISABLED)
```

Pylint (3.3.7)

[convention] Line too long (113/100)

(C0301)